### PR TITLE
add prometheus exporter support

### PR DESCRIPTION
**Pull request by bugVanisher**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -168,6 +168,17 @@ Entering interactive mode (type "help" for commands, "o" for options)
 (pprof) top
 ```
 
+## Exporter
+If you are not satisfied with the build-in web monitor in Locust, you can run prometheus_exporter.py instead of dummy.py as your master.
```
Comment (myzhan) [inline README.md:172]: typo: built-in

File: README.md [hunk 2]
```diff
@@ -168,6 +168,17 @@ Entering interactive mode (type "help" for commands, "o" for options)
...
+
+Try this
+
+```bash
+locust --master -f prometheus_exporter.py
+```
+
+Thanks to Prometheus and Grafana, you will get an awesome dashboard: [Locust for Prometheus](https://grafana.com/grafana/dashboards/12081)
```
Comment (myzhan) [inline README.md:180]: We are missing a guidance about configuring Prometheus to monitor the Locust master.

File: examples/rpc/main.go [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+	"log"
+	"time"
+
+	"github.com/bugVanisher/grequester"
+	"github.com/myzhan/boomer"
+)
+
+var verbose = false
+
+// 修改为要压测的服务接口
```
Comment (myzhan) [inline examples/rpc/main.go:15]: Write comments in EN

Comment (myzhan) [reply examples/rpc/main.go:15]: Can we change to examples/grpc?

File: README.md (hunk 2 above)
Comment (bugVanisher) [reply README.md:180]: I have written a post introduce this, but it is in Chinese.

[locust-with-prometheus-and-grafana](https://bugvanisher.github.io/2020/04/05/locust-with-prometheus-and-grafana/)

File: examples/rpc/main.go (hunk 3 above)
Comment (bugVanisher) [reply examples/rpc/main.go]: > Write comments in EN

done

**Pull request merged**